mod-codex-ekb/src/test/resources/domain/codexekb/basic-junit.feature

```diff
@@ -3,17 +3,20 @@ Feature: mod-codex-ekb integration tests
   Background:
     * url baseUrl
     * table modules
-      | name              |
-      | 'mod-login'       |
-      | 'mod-permissions' |
-      | 'mod-codex-ekb'   |
+      | name                |
+      | 'mod-login'         |
+      | 'mod-permissions'   |
+      | 'mod-codex-ekb'     |
+      | 'mod-codex-mux'     |
+      | 'mod-kb-ebsco-java' |
 
     * table adminAdditionalPermissions
```

Comment:
If this table is empty could it be removed?

Reply:
This table remains empty in all modules. I decided to stick with this